contracts/bprotocol/avatar/Cushion.sol

```diff
@@ -245,4 +249,10 @@ contract Cushion is AvatarBase {
         console.log("amountToLiquidate: %s", amountToLiquidate);
         (amtToDeductFromTopup, amtToRepayOnCompound) = splitAmountToLiquidate(underlyingAmtToLiquidate, amountToLiquidate);
     }
+
+    function quitB() external onlyAvatarOwner() {
+        quit = true;
+        _untop();
```

Comment:
cannot untop when in the middle of liquidation process (even if untop is possible)